File: HW2-Dashboard/src/app/entity/Product.ts
export class Product {

  private _id:number;
  private _name:string;
  private _clientId:number;


  get id(): number {
    return this._id;
  }

  set id(value: number) {
    this._id = value;
  }

  get name(): string {
    return this._name;
  }

  set name(value: string) {
    this._name = value;
  }

  get clientId(): number {
    return this._clientId;
  }

  set clientId(value: number) {
    this._clientId = value;
  }
}

File: HW2-Product-Microservice/src/main/java/com/consulting/mgt/springboot/hw2/product/model/Product.java
package com.consulting.mgt.springboot.hw2.product.model;

import org.springframework.data.annotation.Id;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Product {

	@Id
	private String id;
	
	private String clientId;
}